src/openrct2/actions/SmallSceneryPlaceAction.hpp

```diff
@@ -373,7 +373,8 @@ DEFINE_GAME_ACTION(SmallSceneryPlaceAction, GAME_COMMAND_PLACE_SCENERY, SmallSce
             footpath_remove_litter(_loc.x, _loc.y, targetHeight);
             if (!gCheatsDisableClearanceChecks && (scenery_small_entry_has_flag(sceneryEntry, SMALL_SCENERY_FLAG_NO_WALLS)))
             {
-                wall_remove_at(_loc.x, _loc.y, targetHeight, targetHeight + sceneryEntry->small_scenery.height);
+                // TODO: Seems like heights are in small coords and should be *8 ?
```

Comment:
`tile_element_height`  and `tile_element_water_height` returns big coordinates. This is fine. Remove comment.

Reply:
Ok, thanks for explaining!